pyplanet/apps/contrib/jukebox/migrations/001_map_folder_change_public_to_visibility.py

```diff
@@ -6,7 +6,7 @@
 
 def upgrade(migrator: SchemaMigrator):
 	try:
-		query = RawQuery(MapFolder, """SELECT * FROM `mapfolder` WHERE public = 1""")
+		query = RawQuery(MapFolder, """SELECT * FROM mapfolder WHERE public = 1""")
```

Comment:
I can not show you a screenshot of the error (because I can not reproduce it, I would have to downgrade first, and do not know how to do it), but the error stated that it cannot compare public = 1 because there is no comparison for bool = integer.
Changing the line to the following worked for me.
`query = RawQuery(MapFolder, """SELECT * FROM mapfolder WHERE public = True""")`

Reply:
Tested with `public = True` on MySQL and it works there as well. Updated the PR.